test/test_get_account_statement.py

```diff
@@ -0,0 +1,63 @@
+import os
+import pytest
+
+import pytransferwise
+
+pytransferwise.api_key = os.getenv('TW_API_KEY')
+# print('Get value of TW_API_KEY: {}'.format(pytransferwise.api_key))
+pytransferwise.environment = "live"
```

Comment:
@EgorBEremeev Thanks for adding the test coverage here!

I think it's a little risky to have tests that run against a production API environment by default.  Could you remove this `live` environment spec from the tests?  (removing the line entirely would be great; calls should default to calling the sandbox endpoints)

Reply:
@jayaddison , this is done